opennms-model/src/main/java/org/opennms/netmgt/model/OnmsAlarm.java

```diff
@@ -1153,4 +1151,37 @@ public Date getAckTime() {
         return m_alarmAckTime;
     }
     
+    /**
+     * <p>getAlarms</p>
+     *
+     * @return a {@link java.util.Set} object.
+     */
+    @XmlTransient
+    @ElementCollection
+    @JoinTable(name = "alarm_situations", joinColumns = @JoinColumn(name = "situation_id"), inverseJoinColumns = @JoinColumn(name = "related_alarm_id"))
+    @Column(name="alarm_id", nullable=false)
+    public Set<OnmsAlarm> getRelatedAlarms() {
+        return m_relatedAlarms;
+    }
+
+    /**
+     * <p>setDetails</p>
+     *
+     * @param alarms a {@link java.util.Set} object.
+     */
+    public void setRelatedAlarms(Set<OnmsAlarm> alarms) {
+        this.m_relatedAlarms = alarms;
```

Comment:
Nit: `this.` is redundant when used with a variable prefixed with `m_`.

Reply:
I'm not a fan of either - will remove.